x-pack/plugins/observability_solution/infra/public/components/asset_details/hooks/use_page_header.tsx

```diff
@@ -62,7 +62,7 @@ export const useTemplateHeaderBreadcrumbs = () => {
   const breadcrumbs: EuiBreadcrumbsProps['breadcrumbs'] =
     // If there is a state object in location, it's persisted in case the page is opened in a new tab or after page refresh
     // With that, we can show the return button. Otherwise, it will be hidden (ex: the user opened a shared URL or opened the page from their bookmarks)
-    location.state || history.length > 1
+    !isEmpty(location.state) || history.length > 1
```

Comment:
Using the Asset Details locator was causing this to wrongly display the return button because its value would be `{}`